shell/browser/api/electron_api_web_request.cc

```diff
@@ -115,6 +116,14 @@ v8::Local<v8::Value> HttpResponseHeadersToV8(
     std::string value;
     while (headers->EnumerateHeaderLines(&iter, &key, &value)) {
       base::Value* values = response_headers.FindListKey(key);
+      if (base::EqualsCaseInsensitiveASCII("Content-Disposition", key) &&
```

Comment:
It is to decode the correct file name encoding because there are some webservers, not like webserver in nodejs, which support other encoding string other than UTF8.

Reply:
OK if needed.